doc/classes/Quaternion.xml

```diff
@@ -99,8 +99,9 @@
 		</method>
 		<method name="get_euler" qualifiers="const">
 			<return type="Vector3" />
+			<param index="0" name="order" type="int" default="2" />
```

Comment:
Can't we have the proper enum type here?

Reply:
I don't think this is supported yet. I'd be happy to be proven wrong, though.